### DISCUSSION TITLE
big size tables station_queue and song_history .. how to delete ?

### DISCUSSION THREAD

**Discussion by ajeje989** [AzuraCast/AzuraCast#5544] [Q&A] [answered]
I have 2 very heavy azura database tables

station_queue and
song_history

their content
has 600 MB.

Is it possible to delete the content on history and the queue without causing damage?
If so, how.

**Comment by ghost** [comment]
Yes
you'll need to log in to the terminal and then log in to MySQL/MariaDB and delete the table contents.
DELETE FROM table_name WHERE condition;
Make a back up of your database
mysqldump -u root -p menagerie1 > menagerie1_db_backup.sql

if you make a mistake and need to restore the database
mysqlimport -u [username] -p[password] [dbname] [backupfile.sql]

**Reply by BusterNeece** [reply]
We'd recommend that you please not advise the general users to directly manipulate their database, as this is an extremely risky thing to do that can result in problems.

**Reply by ghost** [reply]
I was only advising what I would do because the documents don't help
you won't be hearing from me any longer I've removed AzuraCast from my server

**Reply by ItsMitchh** [reply]
That was an extreme jump from 0 to 1000, instead of taking the feedback, you just essentially flipped the table and got really mad. Shame, see ya. 

We do always welcome pull requests but in this ever growing hostile world towards open source, people just want us to do it all the time. From your post history, there's nothing in our repos of you mentioning our documentation lacking in this area.

**Comment by BusterNeece** [comment, marked as answer]
> 

There's a setting in System Settings that controls how long history is retained on the system; if you set that to a shorter time period, it will automatically truncate old history entries from time to time.

**Reply by ajeje989** [reply]
> > 
> 
> There's a setting in System Settings that controls how long history is retained on the system; if you set that to a shorter time period, it will automatically truncate old history entries from time to time.

thx worked .. now 38mb all db
